### PR TITLE
Fixed #72

### LINKED ISSUE #72: Inaccurate toast when cancelling Connect to Shield preference.
After solving #24, the resulting toast is inaccurate. It should instead say something like "Connection with Tecla Shield cancelled"

### PR DESCRIPTION
**Pull request by ankitdaf**
1. Added a string to indicate a manual disconnect
2. Added a toast to display the string when "Connect to Tecla Shield" preference is changed

### CODE REVIEW

**Comment by ijdoc** [comment]
Sorry @ankitdaf I did not realize you had removed the shield_disconnected string. This is not what I meant before. What we need is to show a different toast when the connection is cancelled, but the app can arrive at the location where you are changing the string for reasons other than a cancelled discovery. In those instances, it makes more sense to show "Shield disconnected" than "connection cancelled".

In other words, we still need the shield_disconnected string, it just doesn't make sense to show it when and ONLY if the connection is cancelled. Does that make sense?

**Comment by ankitdaf** [comment]
@jorgesilva I have not removed the shield_disconnected string, it is still present in the xml, as well as in the SwitchEventProvider class. I added the toast on the  "onSharedPreferenceChanged" method, which means that it will be called when the Preference "Connect to Shield" is changed. The shield_disconnected Toast is still displayed through the code in the SwitchEventProvider class
